tests/console/ansible.pm

```diff
@@ -156,8 +160,13 @@ sub run {
     # Test that /home/johnd/README.txt is readable and contains the expanded template
     validate_script_output 'sudo -u johnd cat /home/johnd/README.txt', sub { m/my $arch dynamic kingdom/ };
 
+    # Reboot into new snapshot if we test ansible.community.zypper on transactional system
+    #select_console 'root-console';
+    check_reboot_changes if ($test_zypper && is_transactional);
+    #select_serial_terminal;
+
     # Check that Ed - the command line text edit is installed
-    assert_script_run 'which ed' unless is_transactional;
+    assert_script_run 'which ed' if ($test_zypper);
```

Comment:
Following your previous suggestions, the `unless is_transactional` becomes a `unless is_transactional && !is_microos`. Is that intentional?